index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My Website</title>
+    <style>
+      /* Add CSS styling for the navigation bar and search bar */
+      #navbar {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        background-color: #333;
+        color: #fff;
+        padding: 10px;
+      }
+      #navbar a {
+        color: #fff;
+        text-decoration: none;
+        margin-right: 10px;
+      }
+      #search-container {
+        display: flex;
+        align-items: center;
+        background-color: #fff;
+        padding: 10px;
+        border-radius: 10px;
+      }
+      #search-icon {
+        margin-right: 10px;
+      }
+      #search-input {
+        border: none;
+        padding: 10px;
+        font-size: 16px;
+      }
+    </style>
```

Comment:
Please all these styles were supposed to be in the style.css file. Move them to the Style.Css file.

Reply:
Great job adding a polite review 👍 . I think it will be good if you give the student the reason for asking him to move the styles to the CSS file. 😊 ☺️

[OPTIONAL]I think it will be great if you make use of emojis and Gifs. This will make your code review more human-friendly :alien::robot: